### DISCUSSION TITLE
is there any difference in making mods in ports and making mods in source code?

### DISCUSSION THREAD

**Discussion by Whitengos** [Leather128/LeathersFunkinEngine#191] [Q&A] [answered]
I mean when it comes to adding things, as happens in PsychEngine, that in source code you can put it in the folders that are in assets, but in a port it has to be in the mods folder

**Comment by Whitengos** [comment]
:c?

**Comment by Leather128** [comment, marked as answer]
no difference really, making without source is the intended solution tho
